src/engine/PLConstraintScoreTracker.h

```diff
@@ -0,0 +1,109 @@
+/*********************                                                        */
+/*! \file PLConstraintScoreTracker.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Haoze Wu
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** A general class that maintains a heap from PLConstraint to a score.
+
+**/
+
+#ifndef __PLConstraintScoreTracker_h__
+#define __PLConstraintScoreTracker_h__
+
+#include "Debug.h"
+#include "List.h"
+#include "MStringf.h"
+#include "PiecewiseLinearConstraint.h"
+
+#include <set>
+
+#define SCORE_TRACKER_LOG(x, ...) LOG(GlobalConfiguration::SCORE_TRACKER_LOGGING, "PLConstraintScoreTracker: %s\n", x)
+
+struct ScoreEntry
+{
+    ScoreEntry( PiecewiseLinearConstraint *constraint, double score )
+        : _constraint( constraint )
+        , _score( score )
+    {};
+
+    bool operator<(const ScoreEntry& other ) const
```

Comment:
space before const, space before & and not after

Reply:
whoops, done.